packages/tasit-sdk/src/TasitSdk.test.js

```diff
@@ -46,6 +47,27 @@ describe("Decentraland", () => {
   let marketplace;
   let snapshotId;
   let provider;
+  let estateIds;
+
+  const manaFaucetTo = async (beneficiary, amountInWei) => {
+    mana.setWallet(owner);
+    const mintManaToBuyer = mana.mint(
+      beneficiary.address,
+      amountInWei.toString()
+    );
+    await mintManaToBuyer.waitForNonceToUpdate();
+    await confirmBalances(mana, [beneficiary.address], [amountInWei]);
+  };
+
+  const etherFaucetTo = async (beneficiary, amountInWei) => {
+    const connectedOwner = owner.connect(provider);
+    const tx = await connectedOwner.sendTransaction({
+      // ethers. utils.parseEther("1.0")
+      value: "0x0de0b6b3a7640000",
```

Comment:
Ahhh, ok. We can continue to think about whether there's a clever way around that. Because the app code will likely want a convenience function here too. But the comment that's already in the code will serve as a reminder. No change required in this thread.

Reply:
Resolving this